test/cli.py

```diff
@@ -33,3 +40,92 @@ def test_make_time(self):
         self.assertEqual(make_time(ts, fmt='%Y-%m-%dT%H:%M:%S', begin=True), '2031-04-05T00:00:00')
         self.assertEqual(make_time(ts, fmt='%Y-%m-%dT%H:%M:%S', end=True), '2031-04-05T23:59:59')
 
+
+class TestConsumer(TestCase):
+    def setUp(self):
+        """Prepare test case."""
+        super(TestConsumer, self).setUp()
+
+        sys.argv = [
+            'name', '--debug',
+            '--brokers=serv1,serv2',
+            '--ssl=ssl1=sslv1,ssl2=sslv2',
```

Comment:
what do these options mean?  just for testing?  sslv1 and sslv2 should not be used anywhere.

Reply:
It's just strings for testing.